merkle/Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [package]
 name = "merkletree"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
Let's increment the minor number, I think this will break the current API (at least in the way we use it in `rust-fil-proofs`).

Reply:
Actually I think dig takes care of incrementing the number when doing the release, so this is not a big deal.